src/devtools/store.js

```diff
@@ -3,6 +3,7 @@
 import EventEmitter from 'events';
 import {
   TREE_OPERATION_ADD,
+  TREE_OPERATION_CLEAR_CHILDREN,
```

Comment:
super nit "clear" is another way of saying "remove" I guess, but I wonder if a name like `TREE_OPERATION_RECURSIVE_REMOVE_CHILDREN` would be "clearer" heh heh